rest-api-spec/src/main/resources/rest-api-spec/api/clear_scroll_post.json

```diff
@@ -0,0 +1,16 @@
+{
+  "clear_scroll_post": {
```

Comment:
I wonder if this should be `post_clear_scroll`, just to follow the http-verb-preceding-noun convention (put_template, etc)?  TBH my level of caring is like 10% though :)  @elastic/es-clients any thoughts?